components/about-us/aboutuslogo.vue

```diff
@@ -1,11 +1,11 @@
 <template>
   <div class="header">
-    <div class="flex flex-row justify-center items-center w-full main py-100px">
+    <div class="flex flex-row justify-center items-center w-full main py-24">
       <h1 class="aboutus">About Us</h1>
     </div>
     <div>
       <svg
-        class="waves h-15/100 w-full min-h-100px max-h-150px"
+        class="waves h-15/100 w-full min-h-100 max-h-36"
```

Comment:
Maybe we can reduce the height in About Us title animation part as the waves are not visible in single screen space and scrolling is required.

Reply:
Yep that can be good on it

Reply:
@debarati-06 can you take a look now

Reply:
Yes, it looks great!✨